src/python/dxpy/utils/batch_utils.py

```diff
@@ -115,21 +117,21 @@ def _search_column_id(col_name, header_line):
             return i
     raise Exception("Could not find a column with file IDs for {}".format(col_name))
 
-# Parse the CSV file. Create a dictionary with the input arguments for
+# Parse the TSV file. Create a dictionary with the input arguments for
 # each invocation. Return an array of dictionaries.
 #
 # For each line, except the header:
 #   remove informational columns
 #   create a dictionary of inputs we can pass to the executable
 #
-# Example CSV input:
+# Example TSV input:
 # batch ID, pair1,       pair1 ID, pair2,       pair2 ID
 # 23,       SRR123_1.gz, file-XXX, SRR223_2.gz, file-YYY
 #
-def batch_launch_args(executable, input_json, batch_csv_file):
+def batch_launch_args(executable, input_json, batch_tsv_file):
     header_line = []
     lines = []
-    with open(batch_csv_file, "rb") as f:
+    with open(batch_tsv_file, "rb") as f:
         reader = csv.reader(f, delimiter=str(u'\t'))
         for i, line in enumerate(reader):
```

Comment:
How about
```
with open(batch_tsv_file, "rb") as f:
    reader = csv.reader(f, delimiter=str(u'\t'))    
    header_line = next(reader)
    lines = list(reader)
```
For comments filtering it could be `lines = [line for line in reader if len(line) > 0 and not line[0].startswith('#')]` but let's not complicate the input file with comments in this commit, perhaps it's not even what scientists need.